docs/wymeditor_development/making_releases.rst

```diff
@@ -111,24 +118,15 @@ different files:
 Throughout these instructions, when asked to modify the version string, it is
 meant that the version strings be modified in all of these files.
 
-Remove the ``dev`` build metadata from the version string.
-Also, remove the ``+`` because there is no more build metadata.
-
-While you're doing that, make sure that the version strings are otherwise correct.
```

Comment:
After a release, the future version is labeled `next`. So there's no need for `next+dev` anymore.